extensions/amp-story/1.0/amp-story.css

```diff
@@ -180,6 +181,13 @@ amp-story-page {
   z-index: 0 !important;
 }
 
+.i-amphtml-story-media-query-matcher {
+  height: inherit !important;
```

Comment:
not sure I understand how the iframe gets the size of the `amp-story-page`... inherit will make it have the `amp-story` dimensions, no?